build.gradle

```
@@ -71,6 +78,16 @@ subprojects {
         // which provide support to Java users ('-java' as a suffix).
         final evaluatedSubProjectIsJava = evaluatedSubProject.name.contains('-java')
 
+        //dokka configuration for modules
```

for readability and consistency with other comments in this file, please include adequate horizontal space:

```suggestion
        // dokka configuration for modules
```

---

https://github.com/ably/ably-asset-tracking-android/pull/503/commits/f7fd7a9927ffaa09b7caa0bb4ca925428b0df3df